framework/src/org/checkerframework/common/value/ValueTransfer.java

```diff
@@ -189,6 +193,53 @@ private AnnotationMirror createBooleanAnnotationMirror(List<Boolean> values) {
         return new RegularTransferResult<>(newResultValue, result.getRegularStore());
     }
 
+    private TransferResult<CFValue, CFStore> createNewConditionalResult(
+            TransferResult<CFValue, CFStore> result, List<Boolean> resultValues, Node node) {
+        AnnotationMirror stringVal = createBooleanAnnotationMirror(resultValues);
+        CFValue newResultValue =
+                analysis.createSingleAnnotationValue(
+                        stringVal, result.getResultValue().getUnderlyingType());
+
+        return new ConditionalTransferResult<CFValue, CFStore>(
```

Comment:
This should only be a conditional result of the TransferInput to the visitX method is a conditional result.